R/pkg/inst/worker/daemon.R

```diff
@@ -31,7 +31,15 @@ inputCon <- socketConnection(
     port = port, open = "rb", blocking = TRUE, timeout = connectionTimeout)
 
 while (TRUE) {
-  ready <- socketSelect(list(inputCon))
+  ready <- socketSelect(list(inputCon), timeout = 1)
```

Comment:
The logic here can be tested as below:

With a terminal (terminal A) running the command as below:

```bash
nc -l 7777
```
with another terminal (terminal B) running the codes below in R shell.

```R
inputCon <- socketConnection(port = "7777", open = "rb", blocking = TRUE)  
print(socketSelect(list(inputCon), timeout = 1))
```

This prints `FALSE` as it does not have any data first.

In terminal A, if any character is typed first, this returns `TRUE` in terminal B with another execution.